src/AutoRest.CSharp/Mgmt/Generation/MgmtPlaneLongRunningOperationWriter.cs

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using AutoRest.CSharp.Input;
+using AutoRest.CSharp.Mgmt.Decorator;
+using AutoRest.CSharp.Output.Models;
+using AutoRest.CSharp.Output.Models.Shared;
+using Azure;
+using Azure.Core.Pipeline;
+
+namespace AutoRest.CSharp.Generation.Writers
+{
+    /// <summary>
+    /// This is a very naive version of long running operation writer.
+    /// It's written to support generating containers,
+    /// and should be rewritten later.
+    /// </summary>
+    internal class MgmtPlaneLongRunningOperationWriter
+    {
+        internal void Write(CodeWriter writer, OperationGroup operationGroup, Input.Operation operation, Output.Models.Types.BuildContext<Mgmt.AutoRest.MgmtOutputLibrary> context)
+        {
+            writer.UseNamespace("System");
+            writer.UseNamespace("System.Text.Json");
+            writer.UseNamespace("System.Threading");
+            writer.UseNamespace("System.Threading.Tasks");
+            writer.UseNamespace("Azure.Core");
+            writer.UseNamespace("Azure.Core.Pipeline");
+            writer.UseNamespace("Azure.ResourceManager.Core");
```

Comment:
You don't need to write the namespaces here. You can directly import these types in this file and do typeof(Type).